alws/routers/releases.py

```diff
@@ -120,3 +120,45 @@ async def revert_db_release(
         )
     revert_release.send(release_id, user.id)
     return {"message": "Release plan revert has been started"}
+
+
+@router.delete(
+    '/{release_id}/delete',
+    response_model=release_schema.ReleaseCommitResult,
+)
+async def delete_release(
+    release_id: int,
+    db: AsyncSession = Depends(get_db),
+    user: models.User = Depends(get_current_user),
+):
+    """
+    Delete only a scheduled release
+    """
+
+    async with db.begin():
+        release = (
+            (
+                await db.execute(
+                    select(models.Release).where(
+                        models.Release.id == release_id,
+                        models.Release.status == ReleaseStatus.SCHEDULED,
+                    )
+                )
+            )
+            .scalars()
+            .first()
+        )
+        if release is None:
+            return {
+                'message': (
+                    'There is no scheduled release plant with ID '
+                    f'"{release_id}"'
+                ),
+            }
+        else:
+            await db.delete(release)
```

Comment:
Question, are we fine if we don't check if `current_user` has privileges enough to perform this action?

Reply:
I'd say we need to check that user has permissions to operate on releases, yes

Reply:
@javihernandez @Korulag 
I skimmed over other release endpoints and failed to find examples on how we are checking permissions for them.
As far as I see, everything is done via `Dependecny(get_current_user)`

are you suggesting to add some additional checks for this and other enpoints? 
Or am I missing something?

Reply:
When processing the request, you can get the current user and then add a `can_perform` block like this:
```
if not can_perform(release, user, actions.DeleteRelease.name):
    raise PermissionDenied(
        "User does not have permissions to delete this release"
    )
```
This pattern can be found in many places in our codebase.

Actions are defined in https://github.com/AlmaLinux/albs-web-server/blob/master/alws/perms/actions.py, but I see that there's no `actions.DeleteRelease.name`, so you'd need to create it.

Reply:
And I'd say a release can be deleted by the owner and superusers only.

Reply:
Fixed